tests/manual/image.md

```diff
@@ -3,3 +3,12 @@
 * Two images with CKEditor logo should be loaded.
 * Hovering over image should apply yellow outline.
 * Clicking on image should apply blue outline which should not change when hovering over.
+
+### Deleting
```

Comment:
These tests will need to be moved with the widget API, but it's fine to have them here now.